pkg/sdk/procedures_impl_gen.go

```diff
@@ -0,0 +1,579 @@
+package sdk
+
+import (
+	"context"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
+)
+
+var _ Procedures = (*procedures)(nil)
+
+type procedures struct {
+	client *Client
+}
+
+func (v *procedures) CreateProcedureForJava(ctx context.Context, request *CreateProcedureForJavaProcedureRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *procedures) CreateProcedureForJavaScript(ctx context.Context, request *CreateProcedureForJavaScriptProcedureRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *procedures) CreateProcedureForPython(ctx context.Context, request *CreateProcedureForPythonProcedureRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *procedures) CreateProcedureForScala(ctx context.Context, request *CreateProcedureForScalaProcedureRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *procedures) CreateProcedureForSQL(ctx context.Context, request *CreateProcedureForSQLProcedureRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *procedures) Alter(ctx context.Context, request *AlterProcedureRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *procedures) Drop(ctx context.Context, request *DropProcedureRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *procedures) Show(ctx context.Context, request *ShowProcedureRequest) ([]Procedure, error) {
+	opts := request.toOpts()
+	dbRows, err := validateAndQuery[procedureRow](v.client, ctx, opts)
+	if err != nil {
+		return nil, err
+	}
+	resultList := convertRows[procedureRow, Procedure](dbRows)
+	return resultList, nil
+}
+
+func (v *procedures) ShowByID(ctx context.Context, id SchemaObjectIdentifier) (*Procedure, error) {
+	request := NewShowProcedureRequest().WithLike(id.Name())
```

Comment:
It is possible to implement this using in schema syntax which makes this faster and some problems can be avoided (like this one: #2173). This is not possible for every resource but we should do it for every resource where this is possible.

Reply:
done